pkg/console/operator/sync_v400.go

```diff
@@ -192,20 +192,11 @@ func (co *consoleOperator) sync_v400(updatedOperatorConfig *operatorv1.Console,
 
 func (co *consoleOperator) SyncConsoleConfig(consoleConfig *configv1.Console, consoleURL string) (*configv1.Console, error) {
 	updated := consoleConfig.DeepCopy()
-	// track the URL state in prometheus before we update it
-	if consoleConfig.Status.ConsoleURL != consoleURL {
-		// not using this URL anymore
-		co.consoleMetrics.ConsoleURL.WithLabelValues(consoleConfig.Status.ConsoleURL).Set(0)
-	}
-	if len(consoleURL) != 0 {
-		// only update to new if we have a url
-		co.consoleMetrics.ConsoleURL.WithLabelValues(consoleURL).Set(1)
-	}
-
 	if updated.Status.ConsoleURL != consoleURL {
 		klog.V(4).Infof("updating console.config.openshift.io with url: %v", consoleURL)
 		updated.Status.ConsoleURL = consoleURL
```

Comment:
lets put content of this check to `handleConsoleURLMetric()` as well?

Reply:
pulling this out into the `metrics` package entirely.  I think its a better pattern/abstraction.

Reply:
+1 for moving